manufacturing-ai-suite/wind-turbine-anomaly-detection/deployment-package/timeseries-wind-turbine-values.yaml

```diff
@@ -6,9 +6,6 @@
 ---
 namespace: apps
```

Comment:
Let's change the namespace value to "wind-turbine-anomaly-detection".
Same applies to helm/values.yaml.

Reply:
Can be taken as a separate PR